app/views/presentations/_filter.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_tag presentations_path, method: 'get' do %>
+  <p>
+  <%= label_tag('Filter by Presenter:') %>
```

Comment:
I think i need to look at my vim settings, because my indents often un-indent automatically if I make changes to the line for erb files....it must be interpreting the tags as being at the same level? I'll investigate, because you've caught this a couple times.